packages/syft/src/syft/service/request/request.py

```diff
@@ -206,6 +206,7 @@ def apply(self, context: AuthedServiceContext) -> Result[SyftSuccess, SyftError]
             change_status.applied = True
             self.history.append(change_status)
 
+        self.approval_time = DateTime.now()
```

Comment:
MAYBE: By the PR: https://github.com/OpenMined/PySyft/pull/7742 we also have the notion of denying a request.

We could modify the variable name to be `updated_at` to capture approval and denial?
As the approval time , might be fixed only to approval
cc : @shubham3121  , @IrinaMBejan

Reply:
I talked to @shubham3121 and we will change `approval_time` to `updated_at`